### ISSUE TITLE
Some questions on KLD

### ISSUE THREAD

**Issue by gaosh** [YongfeiYan/Gumbel_Softmax_VAE#1]
In your code the KL divergence is calculated by:
`KLD = torch.sum(qy * (log_qy - 1. / categorical_dim), dim=-1).mean()`
I think, for the ` 1. / categorical_dim`, it should be replaced by the `torch.log(1. / categorical_dim),` otherwise, it is not the KL divergence.

**Comment by YongfeiYan** [comment]
Thanks gaosh, I had that bug fixed.
